src/Razor/test/Microsoft.AspNetCore.Razor.Test.ComponentShim/Microsoft.AspNetCore.Razor.Test.ComponentShim.csproj

```diff
@@ -2,6 +2,7 @@
 
   <PropertyGroup>
     <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
+    <DisablePubternalApiCheck>true</DisablePubternalApiCheck>
```

Comment:
Make conditional on the TFM.

Reply:
On tests?

Reply:
Again, this change shouldn't be necessary given how the root Directory.Build.props uses this property.

Reply:
I'll give it a try